get_history.py

```diff
@@ -0,0 +1,39 @@
+from configuration import Configuration
+from database import Database
+from payment import Payment
+
+import json
+import sys
+
+
+def past_records(input_json: str) -> None:
+    config = Configuration()
+    database = Database(config)
+
+    try:
+        past_records_json = json.loads(input_json)
+    except json.JSONDecodeError as e:
+        raise ValueError("The input data is not in json format.")
+
+    for record in past_records_json.get("payments", []):
+        payer = record.get("payer", "")
+        amount = record.get("amount", "")
+        wallet = record.get("wallet", "")
+        note = record.get("note", "")
+        timestamp = record.get("datetime", "")
```

Comment:
Isn't it mandatory for every record to have all these fields? If yes, then first I would add a function to first validate every record in the history_json to see if it contains all the values, and if every value has the expected format (e.g. amount should be float, wallet should be equal to the one of the available wallets, payer should be equal to one of the available payers).

Reply:
Yes It's a good Idea, I'll make another commit and take care of the point you mentioned.